pytorch_lightning/plugins/rpc_plugin.py

```diff
@@ -0,0 +1,18 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pytorch_lightning.plugins.ddp_plugin import DDPPlugin
+
+
+class RPCPlugin(DDPPlugin):
```

Comment:
Why do you need a specific class for this if you don't change anything?

Reply:
Because there is a cycle imports breaking the doc when I m using PipeRpcPlugin. Couldn't resolve the bug.